packages/config/tests/load/tests.js

```diff
@@ -111,7 +111,7 @@ test('--priorityConfig is merged with priority', async (t) => {
 })
 
 test('--priorityConfig falsy values are ignored', async (t) => {
-  const priorityConfig = { build: { command: '', publish: undefined } }
+  const priorityConfig = { build: { command: undefined, publish: undefined } }
```

Comment:
Expected. Empty strings now do override configuration properties, but `undefined` do not.